lib/cmd/fh3/projects.js

```diff
@@ -122,7 +131,12 @@ function createProject(args, deployEnvironment, cb) {
     templates({ _ : ['projects', templateId] }, function(err, template) {
       if (err) return cb(err);
       if (!template) return cb(i18n._('Template not found: ') + templateId);
-      doCall(template, pollCallback);
+
+      var filteredTemplate = filterSelectedApps(template);
+
+      console.log('**metemplate', filteredTemplate );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)